src/PhpSpreadsheet/Writer/Xlsx/Rels.php

```diff
@@ -208,13 +207,18 @@ public function writeWorksheetRelationships(\PhpOffice\PhpSpreadsheet\Worksheet\
         }
 
         if (($pWorksheet->getDrawingCollection()->count() > 0) || (count($charts) > 0) || $drawingOriginalIds) {
-            $relPath = '../drawings/drawing' . $pWorksheetId . '.xml';
-            $rId = ++$d;
+            $rId = 1;
 
+            // Use original $relPath to get original $rId.
+            // take first. In future can be overriten
+            // (! synchronize with \PhpOffice\PhpSpreadsheet\Writer\Xlsx\Worksheet::writeDrawings)
```

Comment:
Comment took from
\PhpOffice\PhpSpreadsheet\Writer\Xlsx\Worksheet::writeDrawings
And an link to it added to show a correlation between this methods: they both use th same [temporary] way of getting $relPath from $drawingOriginalIds array.